bin/tsi.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
I think we could probably rename this file to `testing-instructions`, `tsi` is kinda confusing.